apps/emqx_ft/src/emqx_ft_schema.erl

```diff
@@ -282,6 +291,15 @@ schema(filemeta) ->
             {segments_ttl, hoconsc:mk(pos_integer())},
             {user_data, hoconsc:mk(json_value())}
         ]
+    };
+schema(command_response) ->
+    #{
+        roots => [
+            {vsn, hoconsc:mk(string(), #{default => <<"0.1.0">>})},
```

Comment:
Nit. I think it doesn't make much sense to use complex versioning here, e.g. `1.0` seems good enough.

Reply:
✅